release_notes/ocp-4-20-release-notes.adoc

```diff
@@ -1968,12 +1968,15 @@ In the following tables, features are marked with the following statuses:
 == Known issues
 
 
+[id="ocp-telco-core-release-known-issues_{context}"]
 
+* On systems with certain AMD EPYC processors, some low-level system interrupts, for example `AMD-Vi`, might contain CPUs in their CPU mask that intersect with CPU-pinned workloads. This behavior is because of the hardware design. These specific error-reporting interrupts are generally inactive and there is currently no known performance impact.(link:https://issues.redhat.com/browse/OCPBUGS-57787[OCPBUGS-57787])
```

Comment:
"On systems using specific AMD EPYC processors," "in the CPU mask that overlap with CPU-pinned workloads." I think overlap sound more technical